components/headerControl/headerControl.module.scss

```
@@ -0,0 +1,19 @@
+@import "themes/antd.tokens";
+
+.container {
+    display: flex;
+    justify-content: center;
+    background-color: $tableHeaderBg;
+    width: 32px;
+    height: 36px;
+    border-bottom: 1px solid $tableHeaderBorder;
```

Use the token [colorSplit](https://github.com/Cromeror/Fanatics-Design-System/blob/f40af4029d19180955c616c980147e509c46781d/themes/color.ts#L89) here!
```suggestion
    border-bottom: 1px solid $colorSplit;
```

---

Updated